devices/stello.js

```diff
@@ -0,0 +1,59 @@
+const fz = {...require('zigbee-herdsman-converters/converters/fromZigbee'), legacy: require('zigbee-herdsman-converters/lib/legacy').fromZigbee};
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const extend = require('zigbee-herdsman-converters/lib/extend');
+const e = exposes.presets;
+const ea = exposes.access;
+
+const stello = {
+    fz: {
+          stello_power: {
+            cluster: 'hvacThermostat',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                if (msg.data.hasOwnProperty('16392')) {
+                    return {stello_power: msg.data["16392"]};
+                }
+				else {
+					return 0;
+				}					
+            },
+		  },		  
+		  stello_energy: {
+            cluster: 'hvacThermostat',
+            type: ['attributeReport', 'readResponse'],
+            convert: (model, msg, publish, options, meta) => {
+                if (msg.data.hasOwnProperty('16393')) {
+                    return {stello_energy: parseFloat(msg.data["16393"]) / 1000};

```

Comment:
Why add the `stello_` prefix? (also for power)

Reply:
I am not very familiar with this kind of code and the possibilities, but this worked. If you have suggestions I can try an edited version of the code to confirm it still works. Thank you.

Reply:
You can just remove all the `stello_` prefixes, this looks cleaner to me.

Reply:
All right it does still work without the stello_ prefixes, I submit a  new pull request.